app/models/salt_event_handlers/minion_start.rb

```diff
@@ -0,0 +1,13 @@
+class SaltEventHandlers::MinionStart
```

Comment:
Added documentation. Are you sure about ```Handlers```? What if we want a class that will handle something completely different?